src/solver/conv_hip_implicit_gemm_grouped_fwd_xdlops.cpp

```diff
@@ -339,6 +339,7 @@ void PerformanceConfigHipImplicitGemmGroupFwdXdlops::HeuristicInit(
 
 bool PerformanceConfigHipImplicitGemmGroupFwdXdlops::SetNextValue(const ProblemDescription& problem)
 {
+#if MIOPEN_BACKEND_HIP && MIOPEN_USE_COMPOSABLEKERNEL
```

Comment:
IIRC `MIOPEN_BACKEND_HIP &&` is not necessary
```suggestion
#if MIOPEN_USE_COMPOSABLEKERNEL
```

Reply:
I guess we have to cleanup this pair - it's widely used.